apps/sms/js/thread_ui.js

```diff
@@ -2513,17 +2512,18 @@ var ThreadUI = {
 
     if (event.target.isPlaceholder) {
       typed = event.target.textContent.trim();
-      this.searchContact(typed, this.listContacts.bind(this));
+      this.searchContact(typed)
```

Comment:
Yeah, it would be great if you add the case with several ```then``` to https://etherpad.mozilla.org/sms-code-style-guidelines as we have a lot of them now and will have more in the future :)